src/main/java/DateTimeParser.java

```diff
@@ -0,0 +1,28 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class DateTimeParser {
+
+    DateTimeFormatter dateFormatter;
+    LocalDate localDate;
+    LocalTime localTime;
+    DateTimeFormatter timeFormatter;
+
+    public DateTimeParser(){
+        this.dateFormatter = DateTimeFormatter.ofPattern("uuuu/MM/dd");
+        this.localDate = LocalDate.now();
+        this.localTime = LocalTime.now();
+        this.timeFormatter = DateTimeFormatter.ofPattern("HH:mm");
+    }
+
+    public String formatCurrentDate(DateTimeFormatter dateFormatter, LocalDate ld){
+        return dateFormatter.format(ld);
+    }
```

Comment:
```suggestion
    public String formatCurrentDate(LocalDate localDate){
        return dateFormatter.format(localDate);
    }
```
This class already has a dateFormatter, you should not be passing one into this method.

Reply:
✅